org-code-javabuilder/lib/src/test/java/org/code/javabuilder/UserProjectFileParserTest.java

```diff
@@ -25,7 +25,6 @@ public void canParseValidFileJson() throws InternalServerError, UserInitiatedExc
     assertEquals(projectFiles.getJavaFiles().size(), 1);
     ProjectFile firstFile = projectFiles.getJavaFiles().get(0);
     assertEquals(firstFile.getFileName(), "HelloWorld.java");
-    assertEquals(firstFile.getFileContents(), expectedCode);
```

Comment:
Should this check still be here, but with the string prepended?

Reply:
I removed it here because the import logic isn't related to `UserProjectFileParser` specifically. I added a test for the import statement in `JavaProjectFile` instead. I could instead check that the fileContents contain the expectedCode?